radish34/docker-compose.yml

```diff
@@ -447,13 +447,13 @@ services:
       - ganache
     volumes: # ensure relative paths are correct if embedding this microservice in another application:
       - ./config/merkle-tree:/app/config # mount point might be different if configuring from another application
-      - ./contracts/contracts:/app/contracts:consistent # required if deploying/compiling contracts from within this service (if CONTRACT_ORIGIN = 'default' or 'compile')
+      - ./contracts/artifacts:/app/build/contracts:consistent # required if deploying/compiling contracts from within this service (if CONTRACT_ORIGIN = 'default' or 'compile')
     ports:
       - 9000:80
     environment:
       BLOCKCHAIN_HOST: ws://ganache
       BLOCKCHAIN_PORT: 8545
-      CONTRACT_ORIGIN: 'compile' # Where to find the contractInstances?
+      CONTRACT_ORIGIN: 'default' # Where to find the contractInstances?
```

Comment:
I'm okay with the change but a little concerned by this statement in the Timber repo. "Importantly, Timber can only interpret smart contract interface json files which have been compiled with solc." So while the test pass (I'm assuming) there might be some unexpected behavior from Timber. https://github.com/EYBlockchain/timber#contract_origin 
Have you looked into the impact switching to truffle will have on Timber? Or asking them directly since we have some connections there?

Reply:
I found that the sentence you quoted from Timber is not completely accurate. What Timber does need is the build artifact JSON files to be formatted as described in this function: https://github.com/EYBlockchain/timber/blob/master/merkle-tree/src/compile.js#L62-L75. That is actually the same format that Truffle uses but is different than the default output format of `0x/sol-compiler`. That is why we had to force the `merkle-tree` service to recompile the files itself when using `0x/sol-compiler` within our `radish34/contracts` build.

`0x/sol-compiler` outputs JSON in this format:
```
{
  compiledOutput:
    abi: "...",
    evm:
      bytecode: "0x..."
}
```

`truffle compile` outputs JSON in this format:
```
{
  abi: "...",
  bytecode: "0x...."
}
```

Reply:
I have looked into the impact of switching to truffle only as far as running our tests and monitoring the timber logs to confirm there are no errors reported. I have not reached out to any of the EY research team to ask them about any unintended consequences

Reply:
Okay, it might be nice to tell them this. A PR to their project with your findings perhaps?

Reply:
@breakpointer I created this PR to update their README: https://github.com/EYBlockchain/timber/pull/12